tests/integrations/test_wandb.py

```diff
@@ -64,19 +63,34 @@ def test_slow(dataset_name, model_name, request):
 
 
 def _to_wandb(model, dataset):
+    run = wandb.init(project="tests")
+
     # verify that the logging of a dataset works
-    dataset.to_wandb()
+    dataset.to_wandb(run)
 
     # verify that the logging of scan results works
     scan_results = scan(model, dataset)
-    scan_results.to_wandb()
+    scan_results.to_wandb(run)
 
     # verify that the logging of test suite results works
     test_suite_results = scan_results.generate_test_suite().run()
-    test_suite_results.to_wandb()
+    test_suite_results.to_wandb(run)
 
     # Verify that the logging of the SHAP explanation charts works.
     explanation_results = explain_with_shap(model, dataset)
-    explanation_results.to_wandb()
+    explanation_results.to_wandb(run)
+
+    assert re.match("^[0-9a-z]{8}$", str(run.id))
+    run.finish()
+
+
+@pytest.mark.parametrize(
+    "dataset_name", ["german_credit_data"],)
+def test_error(dataset_name, request):
+    dataset = request.getfixturevalue(dataset_name)
 
-    assert re.match("^[0-9a-z]{8}$", str(wandb.run.id))
+    with pytest.raises(ValueError) as e:
+        dataset.to_wandb()
+    assert e.match(
+        r"There are currently no active wandb runs available.*"
+    )
```

Comment:
you can also do `pytest.raises(ValueError, match="There are ...")`

Reply:
by the way, no need to add ".*" because this uses `re.search`. Thus the expression will match any portion of the string.